spyder/utils/help/tutorial.rst

```diff
@@ -1,382 +1,324 @@
-=====================================================
+#####################################################
 Spyder: The Scientific Python Development Environment
-=====================================================
+#####################################################
 
-*Spyder* is an Integrated Development Environment (IDE) for scientific
-computing with the Python programming language. It comes with an
-Editor to write code, a Console to evaluate it and view the results at
-any time, a Variable Explorer to examine the variables defined
-during evaluation, and several other facilities to help you
-effectively develop the programs you need as a scientist.
+*Spyder* is an Integrated Development Environment (IDE) for scientific computing, written in and for the Python programming language.
+It comes with an :guilabel:`Editor` to write code, a :guilabel:`Console` to evaluate it and view the results at any time, a :guilabel:`Variable Explorer` to examine the variables defined during evaluation, and several other facilities to help you effectively develop the programs you need as a scientist.
 
 
-This tutorial was originally authored by
-`Hans Fangohr <https://fangohr.github.io/>`__ from the
-University of Southampton (UK),
-and subsequently updated for Spyder 3.3.x by the development team
-(see `Historical note`_ for more details).
+This tutorial was originally authored by `Hans Fangohr <https://fangohr.github.io/>`__ from the University of Southampton (UK), and subsequently updated for Spyder 3.3.x by the development team (see `Historical note`_ for more details).
 
 
 
+=======================
 First steps with Spyder
-#######################
+=======================
+
+This section is aimed at Python and Spyder beginners.
+If you find it too simple, please continue to the `next one <python-beginners-ref_>`__.
 
-This section is aimed at Python and Spyder beginners. If you find it too
-simple, please continue to the `next one <python_beginners_ref_>`_.
 
 Execute a given program
------------------------
+~~~~~~~~~~~~~~~~~~~~~~~
 
-* We are going to use this program as a first example:
+We are going to use run program as a first example:
```

Comment:
You have changed "this" to "run". Did you mean to change "use" to "run" instead?

Reply:
Yes, thanks. I make embarrassing mistakes like that quite a bit when editing, and have a hard time noticing them since I tend to read what I meant, and not what I wrote.

Reply:
Fixed!